.changeset/wild-trains-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": minor
+---
+
+Fix NFTs loading
```

Comment:
To update for a more precise description of the fix, cause that's going to end up in the changelog, and without context it's impossible to know what's going on here. 🙏

Reply:
Fix in 12f372df6